src/metabase/api/common/openapi.clj

```diff
@@ -28,23 +29,26 @@
         (update :properties fix-locations))))
 
 (defn- fix-type
-  "Fix type of params to make it more understandable to Rapidoc."
+  "Change type of params to make it more understandable to Rapidoc."
```

Comment:
should we change the function name too ? :D

Reply:
Nah, it "fixes" the types for rapidoc understanding. And how does it "fix" them? By changing! 😁